test/sample_set_filter_test.py

```diff
@@ -220,3 +220,9 @@ def test_order_of_ops(self):
             }
         ])
         self.assertEqual(len(samples_merge), 206)
+
+    def test_get_sampleset_meta(self):
+        ret = self.serviceImpl.get_sampleset_meta({'sample_set_ref': self.sample_set_2_ref})
+        self.assertEqual(type(ret), list)
+        self.assertEqual(type(ret[0]), str)
```

Comment:
here ret is `[results]`. I think you mean `self.assertEqual(type(ret[0]), list)` and `self.assertEqual(type(ret[0][0]), str)` here?